Stark/Plugins/afk.py

```
@@ -118,7 +118,10 @@ async def send_afk_message(user_id, fst_name):
                 start_offset = ent.offset
                 end_offset = ent.offset + ent.length
                 mention_text = m.text[start_offset:end_offset]
-                chat = await c.get_users(mention_text)
+                try:
+                  chat = await c.get_users(mention_text)
+                except:
+                  pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)